### PR TITLE
Jct 128/add docs on deleting team

### PR DESCRIPTION
**Pull request by arthurwolf**
## Checklist

 - [ ] The **PR Title** explains the impact of the change.
 - [ ] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [ ] If this PR changes development workflow or dependencies, they have been A) automated and B) documented under docs/developer/. All efforts have been taken to minimize development setup breakage or slowdown for co-workers.
 - [ ] If HTTP endpoint paths have been added or renamed, or feature configs have changed, the **endpoint / config-flag checklist** (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
 - [ ] If a cassandra schema migration has been added, I ran **`make git-add-cassandra-schema`** to update the cassandra schema documentation.
 - [ ] **changelog.d** contains the following bits of information ([details](https://github.com/wireapp/wire-server/blob/develop/docs/developer/changelog.md)):
   - [ ] A file with the changelog entry in one or more suitable sub-sections. The sub-sections are marked by directories inside `changelog.d`.
   - [ ] If new config options introduced: added usage description under docs/reference/config-options.md
   - [ ] If new config options introduced: recommended measures to be taken by on-premise instance operators.
   - [ ] If a cassandra schema migration is backwards incompatible (see also [these docs](https://github.com/wireapp/wire-server/blob/develop/docs/developer/cassandra-interaction.md#cassandra-schema-migrations)), measures to be taken by on-premise instance operators are explained.
   - [ ] If a data migration (not schema migration) introduced: measures to be taken by on-premise instance operators.
   - [ ] If public end-points have been changed or added: does nginz need un upgrade?
   - [ ] If internal end-points have been added or changed: which services have to be deployed in a specific order?

### CODE REVIEW

**Comment by julialongtin** [comment]
password for the team? this concept doesn't make sense to me.

**Comment by arthurwolf** [comment]
From @battermann : 

```
I would say
- replace BRIG_HOST with GALLEY_HOST
- replace $PASSWORD is the password for that team with $PASSWORD is the password for the team admin
```

File: docs/src/how-to/administrate/users.rst [hunk 1]
```diff
@@ -584,3 +584,21 @@ Where:
...
+
+You do this by running:
+
+.. code:: sh
+
+   curl -i -s --show-error -XDELETE "$GALLEY_HOST/teams/$TEAM_ID" -H'Content-type: application/json' -d'{"password":"$PASSWORD","verification_code":"$VERIFICATION_CODE"}}'
+
+Where:
+
+* `$GALLEY_HOST` is the base URL for your brig service
```
Comment (julialongtin) [inline docs/src/how-to/administrate/users.rst:601]: GALLEY looking for brig?

Comment (arthurwolf) [reply docs/src/how-to/administrate/users.rst]: solved.

**Comment by arthurwolf** [comment]
@julialongtin now that the typo is fixed, need new/final review. thanks!

**Comment by arthurwolf** [comment]
created a new version of this PR at https://github.com/wireapp/wire-server/pull/3740, deleting this one.

**Pull request closed without merging**